kernel/src/osquery.cpp

```diff
@@ -94,6 +159,15 @@ static int subscribe_to_event(osquery_event_t event, int subscribe) {
   }
 
   switch (event) {
+  case OSQUERY_PROCESS_EVENT:
```

Comment:
Do you still need the "TODO" above: https://github.com/mofarrell/osquery/blob/process-events-table/kernel/src/osquery.cpp#L156?